modules/images/regenerate-existing-images/helper.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Functions used by the background process API.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Creates a background job.
+ *
+ * Job refers to the term and queue refers to the `background_job` taxonomy.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $name Name of job identifier.
+ * @param array  $data Data for the job.
+ * @return Perflab_Background_Job|WP_Error Job object if created successfully, else WP_Error.
+ */
+function perflab_create_background_job( $name, array $data = array() ) {
+	// Insert the new job in queue.
+	$term_name = 'job_' . time() . rand();
+	$term_data = wp_insert_term( $term_name, PERFLAB_BACKGROUND_JOB_TAXONOMY_SLUG );
+
+	if ( ! is_wp_error( $term_data ) ) {
```

Comment:
Nit-pick, but it's generally more intuitive to use early returns for the "failure" state, not the "expected" state. In other words, I'd recommend reverting the condition to the opposite and then returning `$term_data` right here.